backend/app/api/api_v1/routers/admin.py

```diff
@@ -228,3 +234,27 @@ async def import_law_policy(
         raise HTTPException(
             status_code=status.HTTP_500_INTERNAL_SERVER_ERROR,
         ) from e
+
+
+@r.put("/documents/{id}", status_code=status.HTTP_200_OK)
+async def update_document(
+    request: Request,
+    id: int,
+    meta_data: DocumentUpdateRequest,
+    db=Depends(get_db),
+    current_user=Depends(get_current_active_superuser),
+):
+    # TODO: As this grows move it out into the crud later.
+    doc_update = update(Document)
+    doc_update = doc_update.values(meta_data.__dict__)
```

Comment:
Why not use the builtin model `dict()` method?

```suggestion
    doc_update = doc_update.values(meta_data.dict())
```

Reply:
This might not scale for the future, because it looks like we'll remove information if we send keys with `null` values... probably ok for now though...